tests/datasets/test_discover.py

```diff
@@ -121,8 +122,8 @@
     ),
     # # No conditions, other referenced columns
     ({"selected_columns": ["group_id"]}, EntityKey.DISCOVER_EVENTS),
-    ({"selected_columns": ["span_id"]}, EntityKey.DISCOVER_TRANSACTIONS),
-    ({"selected_columns": ["group_id", "span_id"]}, EntityKey.DISCOVER),
+    ({"selected_columns": ["span_id"]}, EntityKey.DISCOVER),
+    ({"selected_columns": ["group_id", "span_id"]}, EntityKey.DISCOVER_EVENTS),
```

Comment:
`group_id` is in `DISCOVER_EVENTS`, `span_id` is in the common columns, thus we can hit `DISCOVER_EVENTS`